lib/v2/78dm/index.js

```diff
@@ -0,0 +1,74 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://www.78dm.net';
+    const currentUrl = `${rootUrl}${ctx.path === '/' ? '/news' : /\/[0-9]+$/.test(ctx.path) ? `${ctx.path}.html` : ctx.path}`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+
+    let items = $('.card-title')
+        .slice(0, ctx.query.limit ? parseInt(ctx.query.limit) : 30)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            const link = item.attr('href');
+
+            return {
+                title: item.text(),
+                link: /^https/.test(link) ? link : `https:${link}`,
```

Comment:
This does not work with `/78dm/ht_list/107/0/0/1`
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/11386903/181255178-c979d7d9-0de9-4c4f-9eb8-d906f1605ba5.png)
</details>

Reply:
😅